src/music.ts

```diff
@@ -30,15 +31,15 @@ export class Music {
     };
   }
 
-  static get canonical_notes() {
+  static get canonical_notes(): string[] {
     return ['c', 'c#', 'd', 'd#', 'e', 'f', 'f#', 'g', 'g#', 'a', 'a#', 'b'];
   }
 
-  static get diatonic_intervals() {
+  static get diatonic_intervals(): string[] {
     return ['unison', 'm2', 'M2', 'm3', 'M3', 'p4', 'dim5', 'p5', 'm6', 'M6', 'b7', 'M7', 'octave'];
   }
 
-  static get diatonic_accidentals() {
+  static get diatonic_accidentals(): Record<string, IMusicAccidental> {
```

Comment:
I don't think `IMusicAccidental` is the right type name here, this is really returning an interval->accidental map.

Reply:
@wassertim @mscuthbert any suggestion for the name

Reply:
I did not understand you suggestion either @0xfe

Reply:
@0xfe  I suggest `NotePartsValues`as it contains the same info as `MusicNoteParts` and it was already suggested to remove Music.

Reply:
I will go for the proposed name

Reply:
> I don't think `IMusicAccidental` is the right type name here, this is really returning an interval->accidental map.

It is a map interval->(note,accidental) for that reason I suggested NoteAccidental